### PR TITLE
Rsync all of /mnt/uploads

### PR DESCRIPTION
**Pull request by alexmuller**

### CODE REVIEW

File: modules/govuk/manifests/node/s_asset_master.pp [hunk 1]
```diff
@@ -51,20 +51,22 @@
...
 
-  cron { 'rsync-clean':
+  # FIXME: There may be a couple of directories underneath /mnt/uploads that shouldn't be synced.
+  # Over time we should exclude directories once we know they're not required.
+  cron { 'rsync-uploads':
     user    => 'assets',
     hour    => $copy_attachments_hour,
     minute  => '18',
-    command => '/usr/bin/setlock -n /var/run/virus_scan/rsync-clean.lock /usr/local/bin/copy-attachments.sh /mnt/uploads/whitehall/clean',
+    command => '/usr/bin/setlock -n /var/run/virus_scan/rsync-clean.lock /usr/local/bin/copy-attachments.sh /mnt/uploads',
```
Comment (rjw1) [inline modules/govuk/manifests/node/s_asset_master.pp]: being really pick the lock file name should probably change

Comment (alexmuller) [inline modules/govuk/manifests/node/s_asset_master.pp]: You are correct.

Comment (alexmuller) [inline modules/govuk/manifests/node/s_asset_master.pp]: Updated!

**Comment by surminus** [comment]
YAY I'm a big fan of fixing this :+1:

**Comment by rjw1** [comment]
merge when green

**Pull request merged**